CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_stripe_source v0.9.3
+PR [#](https://github.com/fivetran/dbt_stripe_source/pull/) includes the following updates:
+## Under the Hood:
+- Move the live_mode predicate to the non *_tmp staging models. This is because the [0.4.3 release of fivetran_utils](https://github.com/fivetran/dbt_fivetran_utils/releases/tag/v0.4.3) infers if the source table a package is present, and if not the package will create an empty table to ensure the downstream transformations succeed. Previously the Stripe package wasn't compatible because of where the live_mode predicate existed. In the case there will be an empty data model created, we needed to move the predicate so that the queries and downstream models will still run correctly.
+- Incorporated the new `fivetran_utils.drop_schemas_automation` macro into the end of each Buildkite integration test job.
+- Updated the pull request [templates](/.github).
```

Comment:
Weren't these part of a separate PR? Maybe we should not use the overarching PR reference at the top of the CHANGELOG and instead to individual PR references for each bullet since they are from different PRs.

Reply:
updated!